libs/transactions-journey/e2e-tests/specs/transactions-list.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { expect, TestType } from '@playwright/test';
+import { TransactionFixture } from '../model/transaction';
+
+export function testTransactionsList(
+  test: TestType<TransactionFixture, TransactionFixture>
+) {
+  test.describe(
+    'Transactions list',
+    { tag: ['@e2e', '@transactions', '@transactions-details'] },
+    () => {
+      test.beforeEach(async ({ listPage }) => {
+        await listPage.navigate();
+      });
+
+      test('should display transactions', async ({ listPage, listData }) => {
+        const transactionsNumber = await listPage.getTransactionsNumber();
+        expect(transactionsNumber).toEqual(listData.size);
+      });
+
+      test('should filter transactions', async ({ listPage, listData }) => {
+        for (const expectation of listData.searchExpectations) {
+          await test.step(`Search transactions by "${expectation.term}" term`, async () => {
+            await listPage.searchTransactions(expectation.term);
+          });
+          await test.step(`Validate number of transactions to be ${expectation.count}`, async () => {
+            const transactionsNumber = await listPage.getTransactionsNumber();
+            expect(transactionsNumber).toEqual(expectation.count);
+          });
+        }
+      });
+    }
+  );
+}
+
+export function testTransactionListError(
+  test: TestType<TransactionFixture, TransactionFixture>
+) {
+  test.describe(
+    'Transactions list',
+    { tag: ['@e2e', '@transactions', '@transactions-details'] },
+    () => {
+      test('should display correct error state', async ({ listPage }) => {
```

Comment:
Is there any reason this is separated from the describe block above with the same name?

Reply:
We wanted to showcase that it is possible to have a several test suites, then QAs can choose which of them to run, for example:

in modelbank:
```
testTransactionsListHappyPath
```
locally with mocks
```
testTransactionsListHappyPath
testTransactionsListErrors
testTransactionsListNoData
```

Does it make sense, or would you do this separation using tags ?

Reply:
Or is you comment more about duplicated `Transactions list` name ? Should we just change it for a better visibility ?

Reply:
Decided on the call to use tags :)

Reply:
Updated